splinterd/sample_configs/splinterd-node-0-docker.toml

```diff
@@ -39,3 +39,6 @@ bind = "0.0.0.0:8080"
 ca_certs = "/etc/splinter/certs/generated_ca.pem"
 # db address
 database = "postgres://splinter_admin:splinter_test@127.0.0.1:5432/splinter"
+
+# Toml config version
```

Comment:
Should put this as the first entry in the file (because its a common placement) and change the comment to "Config file format version"

Reply:
Addressed